judges/github-events/github-events.rb

```diff
@@ -223,6 +243,7 @@ def self.fill_up_event(fact, json)
         fact.comment_id = json[:payload][:comment][:id]
         fact.comment_body = json[:payload][:comment][:body]
         fact.who = json[:payload][:comment][:user][:id]
+        skip_event(json) if ignored_bot?(fact)
```

Comment:
@danilax86 Is this code really needed here? It seems to me that checking at the beginning (line 133) will be enough.